osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -290,8 +290,12 @@ private void handleEvent(string line)
             string[] split = line.Split(',');
 
             EventType type;
+
             if (!Enum.TryParse(split[0], out type))
-                throw new InvalidDataException($@"Unknown event type {split[0]}");
+            {
+                Logger.Log($"A beatmap event could not be parsed and will be ignored: {split[0]}", LoggingTarget.Runtime, LogLevel.Important);
```

Comment:
why is it not mentioning type?

Reply:
please read the original exception. please add a message matching it. it should mention that `{split[0]}` is the **type**.